tests/subsys/suit/common/mci_test/mci_test.c

```diff
@@ -233,6 +233,11 @@ int suit_mci_signing_key_id_validate(const suit_manifest_class_id_t *class_id, u
 		return SUIT_PLAT_ERR_INVAL;
 	}
 
+	if (key_id == 0)
+	{
+		return SUIT_PLAT_SUCCESS;
```

Comment:
This change is not acceptable.

Reply:
To my understanding - a proper solution for POSIX-based manifest verification was found and manifest&suit-generator updates can be added to the upmerge PR. @ahasztag : has anything changed wrt the strategy?